app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

```diff
@@ -37,17 +37,19 @@ expander:
     link_url: /non-uk-teachers/non-uk-qualifications
 ---
 
-A postgraduate certificate in education (PGCE) is an academic qualification. You can get this through teacher training.
+After gaining a degree, you may want to go on to complete a postgraduate certificate (PG certificate) or diploma (PG diploma).
```

Comment:
One for policy on the phrasing re PG cert and PG diploma!

Reply:
Also - I find the `After gaining a degree` slightly odd - I was a bit confused about this starting the page as I wasn't sure how this related to teaching. I think the sentence feels a bit weird because it is talking about general postgrad qualifications rather than teaching - like you are just getting a postgrad qualification for the fun of it?

Reply:
```suggestion
If you have a degree and are interested in gaining a qualification in education, you may want to do a postgraduate certificate in education (PGCE).
```

Reply:
Have reworded this, if it makes it more confusing (which I agree!) then it's defo not worth including as they're fringe keywords anyway (as in, general 'useful to have' keywords rather than exact keywords)